python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/load-balance.yml

```diff
@@ -0,0 +1,87 @@
+    keys:
+      policies:
+        type: dict
+        description: Collection of load balancing policy definitions.
+        keys:
+          sand_profiles:
+            type: list
+            description: List of load balancing profiles for SAND-based platforms, used for port-channel and ECMP hashing.
+            primary_key: name
+            items:
+              type: dict
+              keys:
+                name:
+                  type: str
+                  description: Unique name of the load-balancing profile.
+                fields:
+                  type: dict
+                  description: Configure packet fields used as input to the hash function for port-channel and ECMP load balancing.
+                  keys:
+                    udp:
+                      type: dict
+                      description: |-
+                        UDP-specific fields used in the load balancing hash.
+                        Requires EOS version 4.33.1F or higher.
+                      keys:
+                        dst_port:
+                          type: int
+                          required: true
+                          min: 0
+                          max: 65535
+                          description: Use the UDP destination port as a hash input.
+                          convert_types:
+                            - str
+                        payload_bytes:
+                          type: str
+                          description: |-
+                            Specifies the UDP payload bytes to use in hash calculation.
+                            Accepts single bytes (e.g., "10"), comma-separated bytes (e.g., "0,1,5"),
+                            ranges (e.g., "0-15"), or combinations (e.g., "0-10,12,15,20-25").
+                            Valid values are between 0 and 62.
+                          convert_types:
+                            - int
+                        match:
+                          type: dict
+                          description: Configuration to match specific bits and define custom payload-based hashing.
+                          keys:
+                            payload_bits:
+                              type: str
+                              required: true
+                              description: |-
+                                Specifies the bit positions within the UDP payload to match for hashing.
+                                Accepts a single bit (e.g., "12"), a comma-separated list (e.g., "0,3,8"),
+                                a range (e.g., "0-15"), or combinations (e.g., "0-7,9,12-15").
+                                Valid values must be in the range 0 to 503.
+                                Matching is limited to a maximum of 16 bits total.
+                              convert_types:
+                                - int
+                            pattern:
```

Comment:
```
+   load-balance sand profile fsdd
+      fields udp dst-port 666
+         match payload bits 444-447 pattern 0x5 hash payload bytes 44-48,61
+         payload bytes 42-44,54
+   load-balance sand profile fsdd
+      fields udp dst-port 666
+         match payload bits 444-447 pattern 0xc hash payload bytes 44-48,61
+         payload bytes 42-44,54

```

so looks like the patter show up in hexadecimal

Reply:
```
lyv577-RCR...04:47:35(config-s-gm-sand-lb-udp-dst-port-666)#match payload bits 45-55 pattern 0x7d1 hash payload bytes 42
lyv577-RCR...04:47:50(config-s-gm-sand-lb-udp-dst-port-666)#exit
lyv577-RCR...04:47:55(config-s-gm-sand-load-balance-profile-dsadasd)#show session-config diffs
--- system:/running-config
+++ session:/gm-session-config
+load-balance policies
+   load-balance sand profile dsadasd
+      fields udp dst-port 666
+         match payload bits 45-55 pattern 0x7d1 hash payload bytes 42

```

you can :)

Reply:
so no convert from int on this one.